### PR TITLE
vp: Add AYUV/Y210/Y410/P016/Y216/Y416 in DdiVp_GetColorSpaceFromMediaFormat

### PR DESCRIPTION
**Pull request by xhaihao**
…Format

This fixes the pipeline below

GST_GL_PLATFORM=glx gst-launch-1.0 filesrc location=input.444.h265 !
h265parse ! vaapih265dec ! vaapipostproc ! glimagesink

### CODE REVIEW

File: media_driver/linux/common/vp/ddi/media_libva_vp.c [hunk 1]
```diff
@@ -1975,6 +1975,12 @@ VPHAL_CSPACE DdiVp_GetColorSpaceFromMediaFormat(DDI_MEDIA_FORMAT format)
     case Media_Format_422V:
     case Media_Format_P010:
     case Media_Format_IMC3:
+    case Media_Format_AYUV:
```
Comment (FurongZhang) [inline media_driver/linux/common/vp/ddi/media_libva_vp.c:1978]: I personally don't prefer to map these formats as BT601 color space. BT601 is an old color space. Probably, these 10bits, 12bits, 16bits are not BT601.

Comment (FurongZhang) [reply media_driver/linux/common/vp/ddi/media_libva_vp.c:1978]: My question for this issue fix: why not pipeline set the color space explicitly in colour_primaries? For the pipeline, is it failing in DdiCodec_PutSurfaceLinuxHW or DdiVp_ConvertSurface?

Comment (xhaihao) [reply media_driver/linux/common/vp/ddi/media_libva_vp.c:1978]: > I personally don't prefer to map these formats as BT601 color space. BT601 is an old color space. Probably, these 10bits, 12bits, 16bits are not BT601.

I have the similar concern, but it has been set to BT601 for P010 in the driver.

Comment (xhaihao) [reply media_driver/linux/common/vp/ddi/media_libva_vp.c:1978]: how about to return CSpace_BT2020 for 10bits / 12bits / 16bits

Comment (xhaihao) [reply media_driver/linux/common/vp/ddi/media_libva_vp.c:1978]: > My question for this issue fix: why not pipeline set the color space explicitly in colour_primaries? For the pipeline, is it failing in DdiCodec_PutSurfaceLinuxHW or DdiVp_ConvertSurface?

It is falling in DdiCodec_PutSurfaceLinuxHW.

Comment (xhaihao) [reply media_driver/linux/common/vp/ddi/media_libva_vp.c:1978]: @FurongZhang any other comment? Thanks!

Comment (FurongZhang) [reply media_driver/linux/common/vp/ddi/media_libva_vp.c:1978]: @xhaihao Sorry for the late response. May I know which platform you are running? I'd like to try this command by myself. Maybe you know, iHD driver has some default value which is easy to cause issues. So I want to check that. Thanks a lot in advance!

Comment (xhaihao) [reply media_driver/linux/common/vp/ddi/media_libva_vp.c:1978]: I ran the testing on ICL & TGL. @FurongZhang

**Pull request merged**